beacon-chain/cache/llrb.go

```diff
@@ -0,0 +1,581 @@
+// Copyright 2010 Petar Maymounkov. All rights reserved.
```

Comment:
Hi @cyberbono3 can we just import this instead of including the source code in Prysm?

Reply:
@rauljordan I think so ,I just added error handling to LLRB implementation

Reply:
+1, lets not check in this 3rd party code here. can we import the external code? What is the error handling?